app/views/talks/recommendations/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= turbo_frame_tag "recommended_talks" do %>
+  <div data-turbo-temporary data-controller="transition" class="hidden" data-transition-enter-after-value="0">
```

Comment:
TIL [`data-turbo-temporary`](https://turbo.hotwired.dev/handbook/building#preparing-the-page-to-be-cached) 😯